src/defaultsettings.cpp

```diff
@@ -90,7 +90,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("keymap_cinematic", "");
 	settings->setDefault("keymap_toggle_hud", "KEY_F1");
 	settings->setDefault("keymap_toggle_chat", "KEY_F2");
-	settings->setDefault("keymap_toggle_force_fog_off", "KEY_F3");
```

Comment:
Perhaps we should add backwards compatibility here?

Reply:
what do you suggest?

Reply:
Actually forget that question/request. The code to implement it would however be something like this
```C++
KeyPress foggy_setting = getKeySetting("keymap_toggle_fog_enabled");
if (foggy_setting == KEY_F3 && g_settings->exists("keymap_toggle_force_fog_off"))
	foggy_setting = getKeySetting("keymap_toggle_force_fog_off");
```

Moreover, please name the new setting `keymap_toggle_fog` for consistency with `hud` and `chat`.

Reply:
ah ok